src/PIL/Image.py

```diff
@@ -1771,8 +1771,9 @@ def resize(self, size, resample=NEAREST, box=None):
            one of :py:attr:`PIL.Image.NEAREST`, :py:attr:`PIL.Image.BOX`,
            :py:attr:`PIL.Image.BILINEAR`, :py:attr:`PIL.Image.HAMMING`,
            :py:attr:`PIL.Image.BICUBIC` or :py:attr:`PIL.Image.LANCZOS`.
-           If omitted, or if the image has mode "1" or "P", it is
-           set :py:attr:`PIL.Image.NEAREST`.
+           Default filter is :py:attr:`PIL.Image.BICUBIC`.
+           If the image has mode "1" or "P", it is
+           always set :py:attr:`PIL.Image.NEAREST`.
```

Comment:
```suggestion
           always set to :py:attr:`PIL.Image.NEAREST`.
```
I realise you're just copying what was always there.